es-app/src/views/gamelist/ISimpleGameListView.cpp

```diff
@@ -101,12 +102,12 @@ bool ISimpleGameListView::input(InputConfig* config, Input input)
 			return true;
 		}else if(config->isMappedTo("b", input))
 		{
+			Sound::getFromTheme(getTheme(), getName(), "back")->play();
```

Comment:
This is a change in the feature set - can you elaborate? It seems we're now adding the "back" sound when changing from the system view to the carousel.

Reply:
From the game* view to the system* view.